src/image_gen/http_image_gen_calculator.cc

```diff
@@ -46,7 +46,7 @@ using ImageGenerationPipelinesMap = std::unordered_map<std::string, std::shared_
 const std::string IMAGE_GEN_SESSION_SIDE_PACKET_TAG = "IMAGE_GEN_NODE_RESOURCES";
 
 static bool progress_bar(size_t step, size_t num_steps, ov::Tensor&) {
-    SPDLOG_LOGGER_DEBUG(llm_calculator_logger, "Image Generation Step: {}/{}", step, num_steps);
+    SPDLOG_LOGGER_DEBUG(llm_calculator_logger, "Image Generation Step: {}/{}", step + 1, num_steps);
```

Comment:
this is not correct - when using steps=20, there will be 21 iterations:

1/21
2/21
..
20/21
21/21

I think we should ignore first iteration, and print `step, num_steps-1`
this way we will have

1/20
2/20
...
19/20
20/20

Reply:
![image](https://github.com/user-attachments/assets/7c25935b-2dab-4b0a-9414-9a2f133b068a)
Its fine